graph/telemetry/istio/appender/appender.go

```diff
@@ -186,6 +188,12 @@ func ParseAppenders(o graph.TelemetryOptions) (appenders []graph.Appender, final
 		}
 		appenders = append(appenders, a)
 	}
+	if _, ok := requestedAppenders[WaypointAppenderName]; ok || o.Appenders.All {
+		a := WaypointAppender{
+			GraphType: o.GraphType,
+		}
```

Comment:
```suggestion
                waypoints := defaultWaypoints  // define to true above
                waypointsString := params.Get("waypoints")
              	if waypointsString != "" {
        		var waypointsErr error
	        	waypoints, waypointsErr = strconv.ParseBool(waypointsString)
		        if waypointsErr != nil {
			        BadRequest(fmt.Sprintf("Invalid waypoints param [%s]", waypointsString))
        		}
	        }
        	a := WaypointAppender{
	        	Waypoints: waypoints,
        	}
```

`GraphType` is not currently used by the appender and can be removed.  Instead, add a `Waypoints` prop.